components/DBTableMocker.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+    namespace Components;
+
+    use DBQueries\Query;
+
+    /**
+     * Class which is used for mocking Db Tables which allows proper testing
+     */
+    class DBTableMocker {
+        public function getTableDescription(string $tableName):array {
+            $connection = DBConnectionProvider::getConnection(IDBConnection::class);
+
+            $query      = new Query("DESCRIBE `$tableName`");
```

Comment:
Might have to introduce DescribeQueryBuilder class for such cases if we stick to the idea that Query objects are not created by themselves.